website/content/docs/release-notes/v0_15_0.mdx

```diff
@@ -231,6 +252,24 @@ description: |-
     </td>
   </tr>
 
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+    0.15.0
+    <br /><br />
+    (Fixed in 0.15.3)
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    Boundary Desktop does not update
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    If you used Homebrew to install Boundary Desktop on macOS, the Desktop client displays a prompt to update automatically, but it never actually updates.
+    <br /><br />
+    This issue is fixed in version 0.15.3.
```

Comment:
for the desktop client the fix is in the version`2.0.2` and it does use the `0.15.3` binary for the cli.